### PR TITLE
Add Nova View & Copy Path Buttons

### LINKED ISSUE #87: Nova View & Copy Path Buttons
Similar to how WordPress handles "View" buttons on Posts, I would like for all of the resources in this package to have a "View" link in both the Nova Index and the Resource Show pages that is a simple ahref link to the path accessor of the resource. Here is a screenshot of what WordPress does:

Screen Shot 2021-04-17 at 9 46 38 AM

YouTube takes it further and does a "Copy" button for the path to the resource that would be great on the resource Show page with the "View" Button next to it and the path as in their example:

Screen Shot 2021-04-17 at 9 40 57 AM

### PR DESCRIPTION
**Pull request by chx2**
**Page**

- [x] Add 'view' link to resource index entries
- [x] Add copyable link to show resource screen

**Post**

- [x] Add 'view' link to resource index entries
- [x] Add copyable link to show resource screen

**Series**

- [x] Add 'view' link to resource index entries
- [x] Add copyable link to show resource screen

**Topic**

- [x] Add 'view' link to resource index entries
- [x] Add copyable link to show resource screen

Closes #87 

@drewroberts I added series and topic as well. Let me know if I don't need to make buttons for them for whatever reason.

### CODE REVIEW

File: src/Nova/Post.php [hunk 1]
```diff
@@ -28,6 +28,15 @@ class Post extends BaseResource
 
     public static $group = 'Website Blog';
 
+    public function actions(Request $request)
+    {
+        return [
+            (new Actions\PreviewPost($this->id))
+                ->onlyOnTableRow()
+                ->withoutConfirmation(),
```
Comment (chx2) [inline src/Nova/Post.php:36]: It is possible to require a confirmation before redirecting. The default behavior of a Nova actions would normally do so but I tried to mimic how WordPress directly links to a page without question.

File: src/Nova/Post.php [hunk 2]
```diff
@@ -52,7 +61,6 @@ public function fields(Request $request)
             Slug::make('Slug')->from('Title'),
             nova('layout') ? BelongsTo::make('Layout', 'layout', nova('layout'))->nullable() : null,
             DateTime::make('Published', 'published_at'),
-            nova('layout') ? BelongsTo::make('Layout', 'layout', nova('layout'))->nullable() : null,
```
Comment (chx2) [inline src/Nova/Post.php:55]: This change is unrelated to the topic of the PR, but I noticed the layout field was repeated twice in the resource.

**Comment by chx2** [comment]
Screen Shot 2021-04-17 at 2 23 21 PM

@drewroberts by setting these up as actions, the view inside a resource is a little wonky. If we want to have a 'preview' link there, it's generated at the top and requires selecting of the 'view' action to perform the redirect. 

While I do intend on making a dedicated nova field for the copyable link, it will be a less obvious to find the 'preview' link when viewing the individual resource.

File: src/Nova/Post.php (hunk 1 above)
Comment (drewroberts) [reply src/Nova/Post.php:36]: Let's have it open in a new tab. No confirmation needed.

File: src/Nova/Post.php (hunk 2 above)
Comment (drewroberts) [reply src/Nova/Post.php:55]: Nice catch.

**Comment by drewroberts** [comment]
I don't think we need a "Preview" link, let's just do the "View" links.

File: composer.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
     ],
     "require": {
         "php": "^7.3|^8.0",
+        "sixlive/nova-text-copy-field": "^1.5",
```
Comment (chx2) [inline composer.json:21]: Screen Shot 2021-04-18 at 11 31 01 PM

**Comment by chx2** [comment]
I have this pretty much ready to go, in terms of how to predict urls, how is that done? I made the assumption that

- Page has no slug besides the actual page itself
- Blog has 'blog' prefix
- Series has 'series' prefix
- Topic has 'topic' prefix

**Comment by drewroberts** [comment]
All of these models have a path attribute that should be used.

File: composer.json (hunk 3 above)
Comment (drewroberts) [reply composer.json:21]: Can you make the url clickable and open a new tab? That copy icon is great.

Comment (chx2) [reply composer.json:21]: I will probably need to make a custom fork of the repo I imported to pull this off. Doesn't appear like it supports `asHtml` right now but I can probably add that in via my own forked version of it.

Comment (drewroberts) [reply composer.json:21]: That sounds good. We can add a forked version of it later to this package.

**Comment by drewroberts** [comment]
@chx2 Is this ready to merge?

**Comment by chx2** [comment]
> @chx2 Is this ready to merge?

Minus my notes in Slack, yes. I can work on outstanding items in another PR

**Review by drewroberts — APPROVED** [review]
Thank you! 📺

**Pull request merged**